modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/ActionButton/index.js

```diff
@@ -107,7 +115,8 @@ const ActionButton = ({
 		handleAlertStatus,
 		handleRedirectPage,
 		handleDeactivatePage,
-		productName
+		productName,
+		...provisioningResults
```

Comment:
This is strange, you are copying all the array values to parameter of this function: getActivationKeysActionsItems

Look at this function:

```js
const activationKeysActionsItems = getActivationKeysActionsItems(
		productName,
		...provisioningResults
	);
```

I removed some parameters to be easier to explain.

Imagine `provisioningResults` is an array, with many items

When you spread the array items to a function call, this is not a bad practice, but I think is not ideal for this case.

Next:

```js
function getActivationKeysActionsItems(productName, provisioningA, provisioningB, provisioningC,....) {
    // Since you passed an array with spread (...) to this function, to catch all the parameters you need to name, each parameter expected, as I did with provisioningA,B,C...
}
```

or:

```js
function getActivationKeysActionsItems(productName, ...provisioning) {
   Now provisioning is an array, with all the parameters you used spread
}
```

To simplify, you just need to pass a boolean to here, it solves you need



```suggestion
		...provisioningResults
```